### DISCUSSION TITLE
`v16.1.0-experimental-stream-defer.6` fails to work

### DISCUSSION THREAD

**Discussion by saihaj** [graphql/defer-stream-wg#27] [Q&A] [answered]
In `v16.1.0-experimental-stream-defer.6` it doesn't provided `defer` and `stream` as part of `specifiedDirectives` https://github.com/graphql/graphql-js/blob/bc6b2e47512ee11c01eb5185d184990e743df736/src/type/directives.ts#L260-L269

You can check the [CI job](https://github.com/contra/graphql-helix/runs/4494472336?check_suite_focus=true) for https://github.com/contra/graphql-helix/pull/220 where introspection will fail.

**Comment by robrichard** [comment, marked as answer]
@saihaj defer and stream will only be enabled if you pass `enableDeferStream: true` to GraphQLSchema. See additional discussion here: https://github.com/robrichard/defer-stream-wg/discussions/12
